### PR TITLE
Updated Jakarta EE Trademark Compatibility License Agreement to 1.1

### PR DESCRIPTION
**Pull request by linkfang**
Signed-off-by: Zhou Fang <zhou.fang@eclipse-foundation.org>

### CODE REVIEW

**Comment by linkfang** [comment]
waiting for HelloSign's update

**Comment by linkfang** [comment]
Got +1 from Sharon, and can I have the code review first? Will need to merge it later after the hellosign change is live, so marked it as draft for now.

https://gitlab.eclipse.org/eclipsefdn/infrazilla/-/issues/645#note_796398

**Review by chrisguindon — APPROVED** [review]
LGTM - Let's merge once we get the go from Sharon/Shawn.

**Pull request merged**
